apps/molgenis-components/src/components/layout/MolgenisSignIn.vue

```diff
@@ -26,131 +22,104 @@
     </template>
     <template v-slot:footer>
       <ButtonAlt @click="cancel">Cancel</ButtonAlt>
-      <ButtonSubmit form="signin-form">Sign in</ButtonSubmit>
+      <ButtonAction @click="signin">Sign in</ButtonAction>
     </template>
   </LayoutModal>
 </template>
 
 <script>
+import ButtonAction from "../forms/ButtonAction.vue";
 import ButtonAlt from "../forms/ButtonAlt.vue";
 import InputString from "../forms/InputString.vue";
 import InputPassword from "../forms/InputPassword.vue";
 import MessageError from "../forms/MessageError.vue";
 import LayoutForm from "../forms/FormMolgenis.vue";
 import LayoutModal from "./LayoutModal.vue";
-import ButtonSubmit from "../forms/ButtonSubmit.vue";
+import Spinner from "./Spinner.vue";
+
+import { request } from "graphql-request";
 
 export default {
-  name: "SignInForm",
+  name: "MolgenisSignInForm",
   components: {
+    ButtonAction,
     ButtonAlt,
     InputPassword,
     InputString,
     MessageError,
     LayoutForm,
     LayoutModal,
-    ButtonSubmit,
-  },
-  props: {
-    show: {
-      type: Boolean,
-      required: false,
-      default: () => false,
-    },
-    axiosClient: {
-      type: Object,
-      required: false,
-    },
+    Spinner,
   },
   data: function () {
     return {
       email: null,
       password: null,
+      loading: false,
       error: null,
       success: null,
     };
   },
   methods: {
-    async signin() {
-      if (!this.email || !this.password) {
+    signin() {
+      if (this.email == null || this.password == null) {
         this.error = "Email and password should be filled in";
       } else {
         this.error = null;
-
```

Comment:
how does this compare to the 'request' client? would we need to use axios because of ssr?

Reply:
yes, why would you change this ? this PR is not moving things forward